index.html

```diff
@@ -16,6 +15,11 @@ <h1>Tanja Süß</h1>
       <a href="#biography">Biography</a>
     </nav>
   </header>
+  <body>
```

Comment:
This looks like an accidental mistake: we only have one `body` element in an html document.